### PR TITLE
220011 blog translation to Japanese

### PR DESCRIPTION
**Pull request by una-tapa**
Catching up with the blog posts.  

I will run by the local team to get their input before asking Kin's approval. I am also planning to work with the admin (Usually I ask Harry) to get their access to this repo since I cannot tag them.  

For easier review: 
https://github.com/OpenLiberty/blogs/blob/aaad1627d3c70665eafac45b939f34fef9c9ce20/posts/ja/2022-10-25-22.0.0.11-ja.adoc

### CODE REVIEW

File: posts/ja/2023-01-30-22.0.0.11_ja.adoc [hunk 1]
```diff
@@ -0,0 +1,467 @@
+---
+layout: post
+title: "Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11"
```
Comment (kinueng) [inline posts/ja/2023-01-30-22.0.0.11_ja.adoc:3]: `title` needs to be translated from `en` to `ja`

File: posts/ja/2023-01-30-22.0.0.11_ja.adoc [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: Open Liberty 22.0.0.11 での Java SE 19 および分散セキュリティー・キャッシュのサポート - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
+blog_description: Open Liberty 22.0.0.11 provides support for Java SE 19 and distributed security caches which can improve performance and failure recovery.  Also included are improvements to the BELL feature along with CVE and bug fixes.
```
Comment (kinueng) [inline posts/ja/2023-01-30-22.0.0.11_ja.adoc:10]: `seo-description` and `blog_description` need to be translated from `en` to `ja`

**Review by kinueng — CHANGES_REQUESTED** [review]

File: posts/ja/2023-01-30-22.0.0.11_ja.adoc (hunk 1 above)
Comment (kinueng) [reply posts/ja/2023-01-30-22.0.0.11_ja.adoc:3]: Hi @una-tapa, I have used machine translation for the suggested change below. Can you help me double check if the translation is appropriate?
```suggestion
title: "Open Liberty での Java SE 19 および分散セキュリティー・キャッシュのサポート 22.0.0.11"
```

File: posts/ja/2023-01-30-22.0.0.11_ja.adoc (hunk 2 above)
Comment (kinueng) [reply posts/ja/2023-01-30-22.0.0.11_ja.adoc:10]: Hi @una-tapa, I have used machine translation for the suggested change below. Can you help me double check if the translation is appropriate?
```suggestion
seo-description: Open Liberty 22.0.0.11 は、 Java SE 19 および分散セキュリティー・キャッシュをサポートします。これにより、パフォーマンスと障害のリカバリーが改善されます。  また、 CVE およびバグ修正と共に BELL フィーチャーの改善も含まれています。
blog_description: Open Liberty 22.0.0.11 は、 Java SE 19 および分散セキュリティー・キャッシュをサポートします。これにより、パフォーマンスと障害のリカバリーが改善されます。  また、 CVE およびバグ修正と共に BELL フィーチャーの改善も含まれています。
```

**Comment by una-tapa** [comment]
Thank you, @kinueng for catching where I forgot to translate. The translation looks very good.

**Pull request merged**